src/routes/safe/components/Transactions/TxList/Filter/index.tsx

```diff
@@ -230,7 +246,7 @@ const Filter = (): ReactElement => {
                             label="To"
                             type="date"
                             control={control}
-                          />
+                          /> */}
```

Comment:
Could be disabled + tooltip "Coming soon"

Reply:
What do you think of this? 
![image](https://user-images.githubusercontent.com/20442784/172343884-714772c5-0fd7-4183-b5d6-ff012f488823.png)

Reply:
Tooltip seems a bit odd imo
![image](https://user-images.githubusercontent.com/20442784/172344488-4a57379a-d26b-4a61-8582-dab731874661.png)

Reply:
![clock](https://user-images.githubusercontent.com/94694401/172345570-d5ecf650-39fb-4dfc-894f-2c8e002d4176.svg)
how about clock icon on the left and on hover of the icon - coming soon?

Reply:
+ disabled field

Reply:
and no green border highlight

Reply:
I've implemented the following:
![image](https://user-images.githubusercontent.com/20442784/172363890-524aa6fe-b56c-4ac8-81b1-0566c37dc8b8.png)

Reply:
After going through this with @usame-algan, I've adjusted the logo to an hourglass:
![image](https://user-images.githubusercontent.com/20442784/172584605-f924a978-7ba1-419b-9267-f5ad9b960a33.png)